test_scripts/WebEngine/028_OnExitApplication_unregister.lua

```diff
@@ -0,0 +1,49 @@
+--[[ General test configuration ]]
+config.defaultMobileAdapterType = "WS"
```

Comment:
@mvorobio I would propose to create the folder related to proposal WebEngineProjection inside WebEngine folder and move this script there:
 `test_scripts/WebEngine/028_OnExitApplication_unregister.lua`
=>
`test_scripts/WebEngine/WebEngineProjection/028_OnExitApplication_unregister.lua`

Reply:
@aderiabin Please check https://github.com/mvorobio/sdl_atf_test_scripts/pull/1/commits/285c8094c242f73fd09d3a3d348e61f456a7e233